### DISCUSSION TITLE
Handling of multi media deletion on feed of different sync statuses

### DISCUSSION THREAD

**Discussion by savvasdalkitsis** [savvasdalkitsis/uhuruphotos-android#232] [Q&A]
When you press delete on a photo in the Lightbox, you are in one of the three following scenarios:

1) Item is only present on your phone
2) Item is only present on LibrePhotos server
3) Item is on both the server and the phone

I handle those scenarios differently in the latest version of the app, showing a dialog that is either:

1) Delete this item? (on Android 13 I skip this dialog and show the required native permission dialog for media deletion)
2) Send this to trash?
3) What would you like to do? 
  a) delete local only, keeping server, 
  b) delete local and trash server, or
  c) cancel

The question is what do you expect the user journey to be when you multi select items in the feed that are a mix of the 3 scenarios above? (if they all happen to be in one of the 3 scenarios, I can just do the same thing i do for the lightbox)

The simplest thing to do (which is in the current app) is not allow the user to delete multiple media if they are not all remote only.

The second option is to only allow deletion if they all fall under the same category

I fear this is unintuitive though as the user might not know why sometimes they see the delete button and why sometimes they dont

**Comment by ippocratis** [comment]
Dont allow delete if remote only
But tbh I'm fine with any option
